docs/installation.rst

```diff
@@ -11,7 +11,46 @@ provided with pulpcore.
 Install ``pulp_rpm``
 --------------------
 
-Users should install from **either** PyPI or source.
+Users should install from **either** PyPI or source or use ansible-pulp installer.
+
+Install by Ansible-pulp
+***********************
+
+With use of ``ansible-pulp`` installer you need to download supportive
+role before you run installer.
+Only Fedora 29+ and CentOS 7 (with epel repository and python36) are supported.
```

Comment:
Can this become part of the previous paragraph or separated from it with a newline please?